README.md

```diff
@@ -33,8 +33,35 @@ This workflow obtains the relevant spack and GitHub Environment information, and
 
 Using the GitHub Environment, it `ssh`s into the deployment environments `spack` instance, and installs the model associated with the repository that called it. It then copies back relevant metadata and creates a versioned GitHub Release in the caller repository, if it is not a `prerelease` deployment.
 
+#### Usage
+
+For supported `spack`-installable ACCESS-NRI models, simply call the `deploy-1-setup.yml` reusable workflow from the given repositories workflow file, as shown below. Don't forget to add required inputs!
+
+```yml
+deploy:
+  uses: access-nri/build-cd/.github/workflows/deploy-1-setup.yml@main
+  with:
+    version: 1.2.3
+  secrets: inherit
+  permissions:
+    contents: write
+```
+
 ### `undeploy-*`
 
+For given `spack` environments, we can also remove deployments. For example:
```

Comment:
I do think it is worthwhile to stress that this is not a workflow users should expect to use directly, that it is part of the automated deployment testing pipeline.

Reply:
Done!